Websites/webkit.org/code-style.md

```diff
@@ -1287,6 +1287,20 @@ signed b; // Uses "signed" instead of "int".
 signed int c; // Doesn't omit "signed".
 ```
 
+[](#types-use-unsigned) Use unsigned types to represent values which cannot be negative.
+
+###### Right:
+
+```cpp
+unsigned manhattanDistance = ...; // https://en.wikipedia.org/wiki/Taxicab_geometry
```

Comment:
Maybe an example that doesn't require a wikipedia article to explain what it is.
unsigned size() const { ... }
int size() const { ... }

Reply:
shouldn't size be using size_t though?